packages/camera/camera_platform_interface/pubspec.yaml

```diff
@@ -3,22 +3,22 @@ description: A common platform interface for the camera plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/camera/camera_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.0.0-nullsafety
+version: 2.0.0
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.3.0-nullsafety.6
-  plugin_platform_interface: ^1.1.0-nullsafety.2
-  cross_file: ^0.3.0-nullsafety
-  stream_transform: ^2.0.0-nullsafety.0
+  meta: ^1.3.0
+  plugin_platform_interface: ^2.0.0
```

Comment:
">=1.0.0 <3.0.0"

(This unfortunately can result in cases where it resolves in existing projects to a non-nullsafe version until packages are upgraded, but it avoids an unpleasant version-lock problem.)

Reply:
Thanks, I was indeed wondering how to tackle this best. I have updated the version constraints.